src/components/ExportDialog.tsx

```diff
@@ -150,6 +160,14 @@ function ExportModal({
             </label>
           </div>
         )}
+        <label>
```

Comment:
prefer to use a horizontal layout for this rather than vertical

Reply:
Sorry I'm not clear here, are you saying the checkboxes should all be on the same horizontal line?

I kept this checkbox consistent with the "Only selected" checkbox above it, which are currently vertical.